pandas/tests/io/parser/test_parsers.py

```diff
@@ -25,6 +28,18 @@
 from .dtypes import DtypeTests
 
 
+@pytest.mark.high_memory
+def test_bytes_exceed_2gb():
```

Comment:
I considered that, but given that the parser tests in general are all constructed to run all of the tests under one of three parser setups (and this only applies to one of them), it seemed like it would be shoe-horning it in just to keep things "pretty". There were a few other locations where one-off tests for the parsers were created because they had very specific functions, so I thought this was closer to the latter case.

There was also text in the documentation that pointed to the old `unittest` style tests being deprecated in favor of normal `pytest`-style tests.

Of course, all that said, I'm happy to change if you disagree.